src/app/deep-read/.gitignore

```diff
@@ -144,6 +144,6 @@ model*/
 *.avi
 *.mp3
 
-# transcript
-core/temp/.*
-core/temp/*json
+# Temporary files for transcript and summary
+core/temp/
+!core/temp/README.md
```

Comment:
Empty files don't get added to git. If we clone and run, there will be an error saying no such directory.

Reply:
I have added readme.md as an exception in last line, that will make sure that folder is created right?

Reply:
👍